src/class_sqlThread.py

```diff
@@ -1,29 +1,41 @@
+"""
+src/class_sqlThread.py
+======================
+"""
 import threading
 from bmconfigparser import BMConfigParser
+
 import sqlite3
 import time
 import shutil  # used for moving the messages.dat file
 import sys
 import os
 from debug import logger
+
 import helper_sql
 import helper_startup
 import paths
 import queues
 import state
 import tr
-
-# This thread exists because SQLITE3 is so un-threadsafe that we must
-# submit queries to it and it puts results back in a different queue. They
-# won't let us just use locks.
+# pylint: disable=attribute-defined-outside-init,protected-access
 
 
 class sqlThread(threading.Thread):
+    """Implement SQL operations in a dedicated thread.
+
+    This thread exists because SQLITE3 is so un-threadsafe that we must submit queries to it and it puts results back
+    in a different queue. They won't let us just use locks.
+
+    This actually only applies for certain deployments, and/or really old version of sqlite. I haven't actually seen it
+    anywhere. Current versions do have support for threading and multiprocessing. I don't see an urgent reason to
+    refactor this, but it should be noted in the comment that the problem is mostly not valid. Sadly, last time I
+    checked, there is no reliable way to check whether the library is or isn't thread-safe."""
```

Comment:
Not sure if this is the right place for this description. Maybe `helper_sql`? Please check [the doc](https://pybitmessage-test.readthedocs.io/en/latest/autodoc/pybitmessage.helper_sql.html).